syft/serde/compression.py

```diff
@@ -75,6 +76,13 @@ def apply_no_compression(decompressed_input_bin) -> tuple:
     return decompressed_input_bin, NO_COMPRESSION
 
 
+scheme_to_compression = {
```

Comment:
Q: Could we move those default values as the first thing in the file (before function definitions)

Reply:
@gmuraru  I wish I could but I need the functions to be defined first before I add them to the dict

Reply:
Yeah :(